.github/workflows/ci.yml

```diff
@@ -60,9 +60,9 @@ jobs:
           # Limit Next.js to 2 CPUs to prevent file descriptor exhaustion. Empty string uses os.availableParallelism()
           NEXT_PARALLELISM: ${{ runner.os == 'macOS' && '2' || '' }}
           GITHUB_AUTH: ${{ secrets.GITHUB_TOKEN }}
-      - run: pnpm release:changelog
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # - run: pnpm release:changelog
```

Comment:
@brijeshb42 disabled this for now as it broke CI https://github.com/mui/material-ui/actions/runs/19836096598/job/56833636030?pr=47405